FunctionalHW9/MaximumOfTwoT1/Main.java

```diff
@@ -0,0 +1,18 @@
+package MaximumOfTwoT1;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        // 1)Max of two integers arguments.
+        List<Integer> integerList = Stream.of(10, -9).toList();
```

Comment:
Please, try to implement calculation of the maximum in one line (small hint, use IntStream)